BlackMarket/app/SignUp/SignUpCardView.swift

```diff
@@ -0,0 +1,211 @@
+      case .showCookiesPolicy:
+        return .none
+      default:
+        return .none
+      }
+    }
+
+    Scope(state: \.nameState, action: /Action.nameChanged) {
+      BMTextFieldDomain()
+    }
+    Scope(state: \.emailState, action: /Action.emailChanged) {
+      BMTextFieldDomain()
+    }
+    Scope(state: \.passwordState, action: /Action.passwordChanged) {
+      BMTextFieldDomain()
+    }
+  }
+}
+
+struct SignUpCardView: View {
+  
+  let store: StoreOf<SignUpCardDomain>
+  
+  var body: some View {
+    WithViewStore(store, observe: { $0 }) { viewStore in
+      Spacer()
+      ZStack {
+        RoundedRectangle(cornerRadius: UI.CornerRadius.medium)
+          .foregroundColor(.white)
+        VStack {
+          Image.logo
+            .padding()
+          VStack(spacing: UI.Padding.verySmall) {
+            BMTextFieldView(
+              store: store.scope(
+                state: \.emailState,
+                action: SignUpCardDomain.Action.emailChanged
+              )
+            )
+            BMTextFieldView(
+              store: store.scope(
+                state: \.nameState,
+                action: SignUpCardDomain.Action.nameChanged
+              )
+            )
+            BMTextFieldView(
+              store: store.scope(
+                state: \.passwordState,
+                action: SignUpCardDomain.Action.passwordChanged
+              )
+            )
+            Button(action: {
+              //
+            }, label: {
+              BMButton(
+                backgroundColor: .black,
+                textColor: .white,
+                title: LocalizedString.SignUpCardView.signupButtonTitle,
+                cornerRadius: UI.CornerRadius.small
+              ).withStyle(.primary)
+            })
+            .padding(.top)
+            Spacer()
+            HStack {
+              Text(LocalizedString.SignUpCardView.policyDisclaimerFirstPart) +
+              Text("\(.init(viewStore.dataPolicyLink))") +
+              Text(LocalizedString.SignUpCardView.policyDisclaimerSecondPart) +
+              Text("\(.init(viewStore.cookiesPolicyLink))")
+            }
+            .frame(maxWidth: .infinity, alignment: .center)
+            .multilineTextAlignment(.center)
+            .font(.subheadline)
+            .foregroundColor(.blackish)
+            HStack {
+              Text(LocalizedString.SignUpCardView.alreadyHaveAccountTitle)
+              Button {
+                // TODO: Go to log in screen
+              } label: {
+                Text(LocalizedString.SignUpCardView.loginButtonTitle)
+                  .foregroundColor(.blue)
+                  .font(.subheadline)
+              }
+            }.padding(.top)
+          }.padding()
+        }
+      }.frame(maxWidth: .infinity, maxHeight: UI.SignUpCardView.height)
+    }
+  }
+}
+
+private extension UI {
+  struct SignUpCardView {
+    static let height: CGFloat = 550
+  }
+}
+
+private extension Constants {
+  enum Path {
```

Comment:
I am not sure about scoping this type of constant to views/domains. You could end up repeating these in other places and the global Constants enum would serve no purpose.